[2025-05-28T12:17:32.021+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_etl.load_weather_task scheduled__2025-05-28T10:55:00+00:00 [queued]>
[2025-05-28T12:17:32.027+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_etl.load_weather_task scheduled__2025-05-28T10:55:00+00:00 [queued]>
[2025-05-28T12:17:32.028+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-28T12:17:32.038+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_weather_task> on 2025-05-28 10:55:00+00:00
[2025-05-28T12:17:32.043+0000] {standard_task_runner.py:60} INFO - Started process 1739 to run task
[2025-05-28T12:17:32.045+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_data_etl', 'load_weather_task', 'scheduled__2025-05-28T10:55:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/weather.py', '--cfg-path', '/tmp/tmphx6k34zf']
[2025-05-28T12:17:32.047+0000] {standard_task_runner.py:88} INFO - Job 63: Subtask load_weather_task
[2025-05-28T12:17:32.087+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_data_etl.load_weather_task scheduled__2025-05-28T10:55:00+00:00 [running]> on host f1bd559e8a8b
[2025-05-28T12:17:32.156+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_etl' AIRFLOW_CTX_TASK_ID='load_weather_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-28T10:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-28T10:55:00+00:00'
[2025-05-28T12:17:32.441+0000] {python.py:201} INFO - Done. Returned value was: Data cuaca berhasil disimpan. 5 rekaman baru, 40 total dalam sejarah
[2025-05-28T12:17:32.458+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_data_etl, task_id=load_weather_task, execution_date=20250528T105500, start_date=20250528T121732, end_date=20250528T121732
[2025-05-28T12:17:32.501+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-28T12:17:32.513+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
